juriscraper/opinions/united_states/state/ariz.py

```diff
@@ -22,7 +22,12 @@ def __init__(self, *args, **kwargs):
 
     def _get_download_urls(self):
         path = '//a[contains(@id , "hypCaseNum")]/@href'
-        return list(self.html.xpath(path))
+        return [
+            href.replace(
+                "http://www.azcourts.gov", "https://opinions.azcourts.gov"
+            )
+            for href in self.html.xpath(path)
+        ]
 
```

Comment:
why did we make this change.  

I see no difference when I test both versions.  

Also - if we are updating Arizona - let's move it to `opinionsitelinear`.